frontend-instructor/src/hooks/useApi.jsx

```
@@ -0,0 +1,17 @@
+import useInstructorAuth from "../context/instructorAuthContext";
+
+export default function useInstructorApi() {
+  const { instructor } = useInstructorAuth();
+
+  const fetchData = async (apiFunction, params) => {
+    console.log(apiFunction, params);
+    let newParams = {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'newParams' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)